src/helpers/search.js

```diff
@@ -1,35 +1,93 @@
+import { STRINGS } from "../helpers/constants";
+import lucene from "lucene"
+import queryString from "query-string";
+
+export const DEFAULTS_FORM_DATA = {
+    article_text: "",
+    year_start: 1892,
+    year_end: 2014,
+    page: 1,
+    pagelen: 20,
+    author: undefined,
+    title: undefined,
+    article_type_article: true,
+    article_type_advertisement: true,
+    author_title: undefined,
+  };
+  
+function add_and_lucene_string(lucene_string, key, val){
+    return ` ${lucene_string.length > 0 ? 'AND' : ''} ${key}:${val}`; 
+}
+
+function add_and_lucene_nested_string(lucene_string, string){
+    return ` ${lucene_string.length > 0 ? 'AND' : ''} (${string})`; 
+}
+
+function add_or_lucene_string(lucene_string, key, val){
+    return ` ${lucene_string.length > 0 ? 'OR' : ''} ${key}:${val}`; 
+}
+  
 export function createCloudsearchQuery(query){
-    let query_string = "q=" + query.q;
+    try {
+        let query_string = "q.parser=lucene&q=";
```

Comment:
We should actually make createCloudsearchQuery use the query-string library (https://www.npmjs.com/package/query-string) to construct a dictionary with the query string, then stringify it. This would be the easiest way to make sure that special characters such as `{` are properly encoded.

Sample code:

```js
import queryString from "query-string";
const query = {"q.parser": "lucene", "q": "publish_date:[1892-01-01T12:00:00Z TO 2015-01-02T12:00:00Z] AND (article_type:article OR article_type:advertisement)"};
return queryString.stringify(query);
```